bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -58,29 +59,43 @@ <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.')
 </div>
 
 <div class="mzp-l-content">
-  {% call feature_card(
-    title=_('Send tabs instantly to your devices'),
-    image_url='firefox/accounts/fxa-send.png',
-    include_highres_image=True,
-    aspect_ratio='mzp-has-aspect-1-1',
-    class='mzp-l-card-feature-right-half mzp-t-firefox'
-  ) %}
-  <p>{{ _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') }}</p>
-  {% endcall %}
+
+  {% if l10n_has_tag('firefox_send_20190420') %}
+    {% call feature_card(
+      title=_('Take total control of the files you share'),
+      image_url='firefox/accounts/fxa-fxsend.png',
+      include_highres_image=True,
+      aspect_ratio='mzp-has-aspect-1-1',
+      class='mzp-l-card-feature-right-half mzp-t-firefox',
+      ga_title='Try Firefox Send'
+    ) %}
+    <p>{{ _('Feel like a secret agent when you send files online, with a private, encrypted link that ‘self-destructs.’') }}</p>
```

Comment:
I just noticed: should these be double quotes around `self destructs` rather than single quotes?

Reply:
Clay says single quotes, so I guess we'll go with that.